.ruby-version

```diff
@@ -1 +1 @@
-2.0.0-p647
+2.4
```

Comment:
Was this supposed to be checked in?

Reply:
no. sorry. I thought I had it in the ignores.
How do I remove it?